src/components/Header/WalletButton.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react'
+import styled from 'react-emotion'
+
+import { GlobalConsumer } from '../../GlobalState'
+import Button from '../Forms/Button'
+import EtherScanLink from '../Links/EtherScanLink'
+import c from '../../colours'
+
+const GuideWrapper = styled('div')`
+  margin-right: 2em;
+  position: relative;
+  & > a,
+  & > div > a {
```

Comment:
Please remove these descendent selectors and create some new styled components here. It will be hard for someone else to know what you are trying to style here and if anyone move these components around the styles could break